float32[] arm_joints
float32[] arm_joints_requested

float32 gripper_position
float32 gripper_position_requested

bool is_real_robot
bool is_in_emergency_stop
bool is_running

bool gripper_object_detected

string STATUS_RUNNING = running
string STATUS_IDLE = idle
string STATUS_PAUSED = pause
string STATUS_ERROR = error
string status
